genesyscloud/group_roles/genesyscloud_group_roles_proxy.go

```diff
@@ -69,6 +70,11 @@ func getGroupRolesByIdFn(_ context.Context, p *groupRolesProxy, roleId string) (
 func updateGroupRolesFn(_ context.Context, p *groupRolesProxy, roleId string, rolesConfig *schema.Set, subjectType string) (*platformclientv2.APIResponse, error) {
 	// Get existing roles/divisions
 	subject, resp, err := p.authorizationApi.GetAuthorizationSubject(roleId, true)
+
```

Comment:
There was not a nil check here.  So if the user entered a role name that was not real, the subject would not be retrieved and the code would fail with a nil pointer